src/PropertyInfo/Extractor/MappingReflectionExtractor.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\PropertyInfo\Extractor;
+
+use OpenEuropa\EPoetry\Type\ContactPersonIn;
+use OpenEuropa\EPoetry\Type\Contacts;
+use OpenEuropa\EPoetry\Type\CreateRequests;
+use OpenEuropa\EPoetry\Type\LinguisticRequestIn;
+use Symfony\Component\PropertyInfo\Extractor\ReflectionExtractor;
+use Symfony\Component\PropertyInfo\Type;
+
+/**
+ * Class MappingReflectionExtractor.
```

Comment:
We should add a description of the scope of this class, what it does, etc.

Reply:
Done.